wagtail/admin/templates/wagtailadmin/chooser/confirm_external_to_internal.html

```diff
@@ -0,0 +1,19 @@
+{% load i18n wagtailadmin_tags %}
+{% trans "Convert to internal link" as convert_str %}
+{% include "wagtailadmin/shared/header.html" with title=convert_str %}
+
+<div class="nice-padding">
+        {% blocktrans %}
```

Comment:
We should try to avoid having much HTML in translation strings if we can avoid it, often translators will try to translate strings like "noreferrer", or might miss out the `<p>` tags.

Could we, at least, convert this into two separate strings for each paragraph?

Reply:
Yep, will break up up a bit